svelte-on-solana/wallet-adapter
# feature:update-documentation

**Changes:**

Closes #6  

**How to test:**

Open the different `README.md` files that contains the repo. Make sure that explanation is clear enough and nothing is missed to explain core and UI setup. 

**Acceptance:**

Internal and external links need to work properly
